src/main/java/controller/BlackJackController.java

```diff
@@ -0,0 +1,77 @@
+package controller;
+
+import domain.game.BlackJackGame;
+import domain.game.GameResult;
+import domain.participant.Command;
+import domain.participant.Name;
+import domain.participant.Participant;
+import java.util.List;
+import view.InputView;
+import view.OutputView;
+
+public class BlackJackController {
+
+    public void run() {
+        BlackJackGame blackJackGame = createBlackJackGame();
+
+        Participant dealer = blackJackGame.findDealer();
+        List<Participant> players = blackJackGame.findPlayers();
+
+        OutputView.printInitialCards(dealer, players);
+
+        playPlayersTurn(blackJackGame, players);
+        playDealerTurn(blackJackGame, dealer);
+
+        showGameResult(blackJackGame, dealer, players);
+    }
+
+    private BlackJackGame createBlackJackGame() {
+        try {
+            List<Name> playerNames = InputView.inputPlayerNames();
+            return new BlackJackGame(playerNames);
+        } catch (IllegalArgumentException e) {
+            OutputView.printException(e);
+            return createBlackJackGame();
+        }
+    }
+
+    private void playPlayersTurn(BlackJackGame blackJackGame, List<Participant> players) {
+        for (Participant player : players) {
+            playPlayerTurn(blackJackGame, player);
+        }
+    }
+
+    private void playPlayerTurn(BlackJackGame blackJackGame, Participant player) {
+        Command command = Command.HIT;
+
+        while (command != Command.STAY && !player.isFinished()) {
+            command = inputCommand(player);
+            blackJackGame.drawCardByCommand(player, command);
```

Comment:
```suggestion
            blackJackGame.drawCard(player, command);
```
파라미터 값으로 전달하는데 메서드명에 또 쓸 필요는 없겠네요.